web_ui/prometheus.go

```diff
@@ -228,6 +228,13 @@ func configDirectorPromScraper(ctx context.Context) (*config.ScrapeConfig, error
 		RefreshInterval:  model.Duration(15 * time.Second),
 		HTTPClientConfig: sdHttpClientConfig,
 	}
+
+	// Cardinality limits for Prometheus, configured via Pelican configuration file
+	scrapeConfig.LabelLimit = uint(param.Monitoring_LabelLimit.GetInt())
```

Comment:
For all of these parameters, what happens if the user sets them to a negative or huge value? Can you do some input validation before passing them to Prometheus?